docs/examples/use_cases/tensorflow/yolov4/src/train.py

```diff
@@ -112,7 +112,8 @@ def dataset_fn(input_context):
 
     multigpu = kwargs.get("multigpu")
     strategy = tf.distribute.MirroredStrategy() if multigpu else tf.distribute.get_strategy()
-    atexit.register(strategy._extended._collective_ops._pool.close) # type: ignore
+    if hasattr(strategy._extended._collective_ops, "_pool"):
+        atexit.register(strategy._extended._collective_ops._pool.close) # type: ignore
```

Comment:
Do we know if no WAR is needed in the new tf version where the old WAR does not work?

Reply:
I guess the passing tests may be the answer, but maybe it is random thingy?

Reply:
🤷 The CI will tell us.
Someone in https://github.com/tensorflow/tensorflow/issues/50487 claims that it works with the latest.

Reply:
> maybe it is random thingy?

It was 100% reproducible before the fix.